DeploymentCloud/Deployment.Common/Resources/Templates/Spark-Template.json

```diff
@@ -139,6 +139,7 @@
 						}
 					]
 				},
+				"minSupportedTlsVersion": "1.2",
```

Comment:
do we need this for other resources as well?

Reply:
Not aware of any other resource that need this

Reply:
other known resources like the website already is in TLS 1.2